lib/gooddata/bricks/hello_world_brick.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'brick'
+
+module GoodData::Bricks
```

Comment:
Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style.